config.yaml

```diff
@@ -49,3 +49,23 @@ options:
     default: ""
     description: |
       A YAML list which will be injected to define additional prospectors/inputs.
+  ignore_older:
+    type: string
+    default: "0"
+    description: |
+      If this option is enabled, Filebeat ignores any files that were modified before the specified timespan. 
+      Configuring ignore_older can be especially useful if you keep log files for a long time:
+      https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-log.html#filebeat-input-log-ignore-older"
+  clean_inactive:
+    type: string
+    default: "0"
+    description: |
+      When this option is enabled, Filebeat removes the state of a file after the specified period of inactivity has elapsed:
+      https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-log.html#filebeat-input-log-clean-inactive"
+  close_timeout:
```

Comment:
the related bug only mentions `clean_inactive`, why do we need to expose all three settings?

Reply:
I should remove `close_timeout` from the patch as it may not necessarily be useful and could cause problems. It was initially added targeting some specific scenarios but overall those are unlikely.

Reply:
`ignore_older` was added as it resolves another issue reported (customer ticket) and allows users also to avoid collecting logs they don't need.

Reply:
logs that aren't needed can be ignored via the `excluded_files` charm option; in any case I would prefer to keep the two PRs separate

Reply:
just `clean_inactive` and `ignore_older` included in the patch as the issue is resolved by setting those, no need for `close_timeout`

Reply:
why do you also need `ignore_older`? Isn't `excluded_files` sufficient for your use-case?

Reply:
@al3jandrosg this conversation is still open: why do you need to configure `ignore_older`? The original bug only ask for tuning `clean_inactive`

Reply:
@aieri `ignore_older` is used by the customer(s) to avoid harvesting logs older than a certain time threshold. `exclude_files` is not precisely what they need. As for the file descriptor addressed in this PR, `ignore_older` helps the mitigation of potential issues using `clean_inactive`, as it ensures that no harvesters are triggered for older logs and the effects of `clean_inactive` i.e., removing the state of the file, are _safer_ (customers have raised concerns about using `close_*` and `clean_*` directives because the Filebeat docs warn about potential data loss side effects).